backend/sozisel/priv/repo/migrations/20210513120646_create_event_results.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Sozisel.Repo.Migrations.CreateEventResults do
+  use Ecto.Migration
+
+  def change do
+    create table(:event_results) do
+      add :result_data, :string
+      add :participant_token, references(:participants, type: :string, column: :token_id, on_delete: :nothing), null: false
```

Comment:
we should probably reference participant_id as mentioned previously. The token is needed just to authorize particpant when he/she sends mutations.